README.md

```diff
@@ -121,42 +132,45 @@ vars:
   hubspot_email_event_delivered_enabled: false
   hubspot_email_event_dropped_enabled: false
   hubspot_email_event_forward_enabled: false
+  hubspot_email_event_click_enabled: false
   hubspot_email_event_open_enabled: false
   hubspot_email_event_print_enabled: false
   hubspot_email_event_sent_enabled: false
   hubspot_email_event_spam_report_enabled: false
   hubspot_email_event_status_change_enabled: false
-
-  hubspot_contact_merge_audit_enabled: true               # Enables the use of the CONTACT_MERGE_AUDIT table (deprecated by Hubspot v3 API) for removing merged contacts in the final models.
-                                                          # If false, ~~~contacts will still be merged~~~, but using the CONTACT.property_hs_calculated_merged_vids field (introduced in v3 of the Hubspot CRM API)
-                                                          # Default = false
+  hubspot_form_enabled: false                             # Disables form metadata and relationships to contacts
```

Comment:
Should this be `hubspot_contact_form_enabled`?

Reply:
Removed in favor of the prior!